pkg/rkt-stage1/Dockerfile.in

```diff
@@ -0,0 +1,21 @@
+# hadolint ignore=DL3006
+FROM KERNEL_TAG as eve-kernel
+
+FROM golang:1.11.11-alpine3.8
+ENV GO111MODULE=on
+
+RUN apk update && apk add bash git make autoconf gcc acl-dev musl-dev musl-utils libelf-dev linux-headers openssl-dev bc patch busybox-static
+
+ADD https://github.com/rkt/stage1-xen/archive/master.zip /tmp
+RUN unzip /tmp/master.zip -d /go
+COPY 0001-Go-12-upgrade.patch /go/stage1-xen-master
+COPY 0002-eve-kernel.patch /go/stage1-xen-master
+RUN cd /go/stage1-xen-master && patch -p1 < 0001-Go-12-upgrade.patch && patch -p1 < 0002-eve-kernel.patch
```

Comment:
Instead of cd please use WORKDIR /go/stage1-xen-master Dockerfile directive

Reply:
Fixed.